### PR TITLE
Correctly locate common proto files

### PR DESCRIPTION
**Pull request by stephenplusplus**
https://github.com/googleapis/gax-nodejs/issues/249#issuecomment-397688582

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,3 +8,4 @@ system-test/secrets.js
 system-test/*key.json
 *.lock
 *-lock.js*
+google/
```
Comment (JustinBeckwith) [inline .gitignore]: add a newline

**Comment by JustinBeckwith** [comment]
@stephenplusplus thank you for doing this 😸

File: .jsdoc.js [hunk 2]
```diff
@@ -23,23 +23,19 @@ module.exports = {
...
-    destination: './docs/'
+    destination: './docs/',
   },
-  plugins: [
-    'plugins/markdown'
-  ],
+  plugins: ['plugins/markdown'],
   source: {
     excludePattern: '(^|\\/|\\\\)[._]',
-    include: [
```
Comment (crwilcox) [inline .jsdoc.js]: the change in formatting here, is this the result of running the prettier?

Comment (stephenplusplus) [reply .jsdoc.js]: Yep yep.

File: load.js [hunk 3]
```diff
@@ -4,39 +4,60 @@ var globby = require('globby');
...
+        .map(path.normalize);
+    }
+
+    return COMMON_PROTO_FILES;
+  }
+
   // Causes the loading of an included proto to check if it is a common
   // proto. If it is a common proto, use the google-proto-files proto.
   resolvePath(_, includePath) {
+    includePath = path.normalize(includePath);
```
Comment (crwilcox) [inline load.js]: is this just for safety? You also normalize when populating COMMON_PROTO_FILES

Comment (stephenplusplus) [reply load.js]: Yeah, globby actually reverses the slashes, and we don't know what's coming into `resolvePath`, so this just makes sure there's a consistent format between the input and the list we made of files on disk.

**Pull request merged**